tools/docker/README.md

```diff
@@ -12,14 +12,17 @@ Usage
 There is a public build of the image:
 <https://hub.docker.com/r/hammerlab/ketrew-server/>.
 
-    docker pull hammerlab/ketrew-server
+    DOCKER_TAG=latest  # Or any other tag
+    docker pull hammerlab/ketrew-server:$DOCKER_TAG
 
```

Comment:
Ugh, DHub's rendering of the markdown is a bit weird https://hub.docker.com/r/hammerlab/ketrew-server/